[feature] 45 project section done

Rahilsiddique opened the pull request:
things done
- [x] carousel 
- [x] Responsive design

riyavishwa981 commented on src/components/ProjectSection/index.js:
```diff
@@ -0,0 +1,145 @@
+  };
+  return (
+    <div className="relative mx-5 my-10 md:my-10">
+      <button className="inline-block px-2 bg-gray-700 md:mx-12 rounded-xl">
```
add text-white

riyavishwa981 commented on src/components/ProjectSection/index.js:
```diff
@@ -0,0 +1,145 @@
+        Developed Projects
+      </button>
+      <div className="flex flex-col my-3 md:gap-3 md:mx-12">
+        <div className="text-xl font-semibold md:text-5xl">
```
add text-white

riyavishwa981 commented on src/components/ProjectSection/index.js:
```diff
@@ -0,0 +1,145 @@
+    const currentNameDesc = tempProjects.filter(
+      project => swiper.activeIndex === tempProjects.indexOf(project)
+    );
+    setNameDes(p => ({
```
Instead of p put a meaningful name so that it can be understood

riyavishwa981 commented:
Everything seems good , the page works fine on mobile screen as well just make the small changes as mentioned

JayeshVP24 commented:
the implementation was very good but little responsive issues.
rounded corners of image are not working properly, my guess is the image or div height has overflew out of the div
the overlay text is not exactly responsive on md: screens.
fixing the image dimensions might solve most issues

u also forgot the badge of  `developed projects`  at the top 
rename titles and content with whatever is in figma file

kindly give project title to sidebar pagination images 

also use the navigations svgs in figma.

the implementation was excellent, nicely done!

Rahilsiddique commented:
- [x] added the project title to the pagination preview section 
- [x] fixed the issues related to responsive design 
- [x]   added the shadow svg 

- caveats
- [ ] unable to use the figma svg icons for navigation
- [ ] unable to edit the dot pagination 
- [ ] responsive design may still need some work 😪

JayeshVP24 reviewed (approved):
LGTM
